app/controllers/authenticated_controller.rb

```diff
@@ -7,7 +7,7 @@
 # Any third-party code (e.g. plugins) should inherit from this class instead of
 # calling the authentication filters directly
 class AuthenticatedController < ApplicationController
-  before_action :login_required
+  prepend_before_action :login_required
```

Comment:
Do you think that it is clear why we're using `prepend_` here? I'm concerned about someone reverting this in the future.